statusboard/src/components/Header/Header.js

```diff
@@ -9,15 +9,15 @@ function Header() {
   // https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/headings.html
   return (
     <header className="App-header">
-      <NavLink to="/" className="home-link">
+      <NavLink to="https://brigade.codeforamerica.org/" className="home-link">
         <img
           src={logo}
           className="App-logo"
           alt="Code for America brigade"
           role="presentation"
         />
-        <span className="App-title">Project Index</span>
       </NavLink>
+      <span className="App-title">Project Index</span>
```

Comment:
**Fix hamburger**: wrap both the `NavLink` and the `span` inside a `div` so that the layout knows they should be seen as one element (and be careful to **not** include the `<Nav />` just below)